components/game/context.tsx

```diff
@@ -65,13 +66,14 @@ export const GameContextProvider = ({
   }, [id]);
 
   const move = useCallback(
-    (coords: Coords) => {
+    (coords: Coords, currentPlayer: Player, gameType: GameType) => {
```

Comment:
Just an observation: 

if we think about this function as being called in the context of one game, that'd mean that during one game, the game type can be changed between turns

Similar thing about the currentPlayer arg: this notation makes it possible that a player would make a move out of their order (i.e. at the state of game A I can call it with both `(x, y, player1)` or `(x, y, player2)`